### PR TITLE
productionify the backend

### PR DESCRIPTION
**Pull request by camenpihor**
- use waitress for production
- remove need for environment variables
- update install instructions

### CODE REVIEW

**Comment by camenpihor** [comment]
@mjhoy the only unreviewed code are in 3821663 and 9cf1fe0

File: backend/.pylintrc [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+# Regular expression matching correct attribute names. Overrides attr-naming-
+# style.
+#attr-rgx=
+
+# Bad variable names which should always be refused, separated by a comma.
+bad-names=foo,
+          bar,
+          baz,
+          toto,
+          tutu,
```
Comment (mjhoy) [inline backend/.pylintrc:306]: haha tutu? is that foo in another language?

File: backend/.pylintrc [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+                  _make
+
+# List of valid names for the first argument in a class method.
+valid-classmethod-first-arg=cls
+
+# List of valid names for the first argument in a metaclass class method.
+valid-metaclass-classmethod-first-arg=cls
+
+
+[DESIGN]
```
Comment (mjhoy) [inline backend/.pylintrc:465]: engineering would fail this section hard

File: backend/.pylintrc (hunk 1 above)
Comment (camenpihor) [reply backend/.pylintrc:306]: Hahaa I have no idea, its a pylint default

File: backend/.pylintrc (hunk 2 above)
Comment (camenpihor) [reply backend/.pylintrc:465]: haha yea... I sometimes bump it up 😬

**Pull request merged**